packages/clerk-js/src/ui/components/SignUp/SignUpStart.tsx

```diff
@@ -315,7 +317,7 @@ function _SignUpStart(): JSX.Element {
             <Card.ActionText localizationKey={localizationKeys('signUp.start.actionText')} />
             <Card.ActionLink
               localizationKey={localizationKeys('signUp.start.actionLink')}
-              to={clerk.buildUrlWithAuth(signInUrl)}
+              to={isCombinedFlow ? '../' : clerk.buildUrlWithAuth(signInUrl)}
```

Comment:
I believe this would break if the future is turned on from the Provider, but the SignUp component is mounted either way. Isn't that the case ?

Reply:
@panteliselef Good catch! We've updated the logic to detect when SignUp is being rendered within SignIn. This should allow users to use `<SignUp />` manually even if they've enabled `combinedFlow` on `ClerkProvider` (which we will recommend against).